packages/nextcloud/lib/src/nextcloud.openapi.dart

```diff
@@ -10,68 +10,11 @@ import 'package:built_value/serializer.dart';
 import 'package:built_value/standard_json_plugin.dart';
 import 'package:dynamite_runtime/content_string.dart';
 import 'package:dynamite_runtime/http_client.dart';
-import 'package:universal_io/io.dart';
 
 export 'package:dynamite_runtime/http_client.dart';
 
 part 'nextcloud.openapi.g.dart';
 
-class NextcloudResponse<T, U> extends DynamiteResponse<T, U> {
-  NextcloudResponse(
-    super.data,
-    super.headers,
-  );
-
-  @override
-  String toString() => 'NextcloudResponse(data: $data, headers: $headers)';
-}
-
-class NextcloudApiException extends DynamiteApiException {
```

Comment:
I'd still like having a separate exception.

An app could also include a nextcloud unrelated openapi schema and this makes it impossible to distinguish them.

Reply:
Why would it? We have stacktraces, right?

Reply:
I was acrually thinking of trying to handle the error.
Imagine hypothetical case where a dev might want to generate two separate specs (dynatime should be general purpose) and the dev might want to handle the errors differently (in a try catch or the UI).

For this they'd need to know what they are dealing with.

This is just dynamite stuff and we'd throw the spec specific one (NewsError, NotesError) but neon could just handle the parent (DynamiteError).

Reply:
Makes sense, although I don't know if anyone will ever have this use-case. As you said for neon we should just handle the generic DynamiteException.